packages/itinerary-body/src/styled.tsx

```diff
@@ -286,6 +268,7 @@ export const LegClickableButton = styled(TransparentButton)`
   position: absolute;
   right: 0;
   top: 0;
+  width: 100%;
```

Comment:
This doesn't do anything more, the width is already set by `left` and `right` under absolute positioning.

Reply:
Removing this causes the button to only be a few pixels wide in Safari